corehq/apps/users/models.py

```diff
@@ -1603,19 +1604,18 @@ def set_has_built_app(self):
             self.has_built_app = True
             self.save()
 
-    def log_user_create(self, created_by, created_via):
+    def log_user_create(self, domain, created_by, created_via, can_skip_domain=False):
```

Comment:
nit: I think a better name for this would be `domain_required_for_log`. That would have to default to `True` but I think it would be clearer, particularly in methods like `WebUser.create` where it's not clear what `can_skip_domain` means.

Reply:
https://github.com/dimagi/commcare-hq/pull/29889/commits/767b8c1af364414f8c03cb6e2c2157dbcabd67a1